bitnami/mean/requirements.lock

```diff
@@ -1,6 +1,9 @@
 dependencies:
 - name: mongodb
   repository: https://kubernetes-charts.storage.googleapis.com/
-  version: 0.4.7
-digest: sha256:baf63f214bb88d0d7100eea86e218159a1d78e602e7e988db9b6ee73d282cb44
-generated: 2017-02-24T17:51:04.928104617+05:30
+  version: 2.0.2
```

Comment:
The latest MongoDB chart version is 2.0.6
Could you update it and check if everything works fine?